keadm/cmd/keadm/app/cmd/util/exec.go

```diff
@@ -18,6 +21,11 @@ type Command struct {
 }
 
 func NewCommand(command string) *Command {
+	if runtime.GOOS == "windows" {
```

Comment:
Would it be better to use exec_windows and exec_others in combination with the changes in GetStdErr() and GetStdOut?

Reply:
done